### ISSUE TITLE
Homepage: Try a variety of headlines to cycle through

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-main-2022#101]
The homepage reads "WordPress: Publish your _passion_". Since WordPress is more than that, it would be good to see a range of options there in a rotating headline: “Publish your ______?” And we can get quirky there. Something like 3-5 good options, with the specific copy to be explored.

As a simple starting point, the headline block does not need to be interactive, and can simply surface hardcoded options:

<img width="1400" alt="mock" src="https://user-images.githubusercontent.com/1204802/185965636-f69dccec-0a3a-4e63-9470-69b84e93aed7.png">

This would presumably simplify ensuring a well translatable and accessible solution, and can grow into bigger things if need be in the future.

Semantically, one option could be to have the first heading be static, the subsequent 2-5 headings to swap out with be fake and, then pair it with a hidden heading announcing the a full range. Something like:

* “Publish your passion”
* “Publish your diary”
* […]

And then in the hidden heading, “Publish your passion, your diary, your […], or your […] with WordPress”. Something to explore and vet in practice.

**Comment by ryelle** [comment]
In slack, we talked about swapping out the whole phrase after "WordPress" so that it would be easier to translate (thinking of languages with different word order, where this would be "your passion publish").

Accessibility-wise, it would be better to stick to one heading and let the animated ones be readable content, the suggested "Publish your passion, your diary, your […], or your […] with WordPress" could get wordy.

**Comment by ryelle** [comment]
I've got a basic block started using the p2 code, it looks like this:

https://user-images.githubusercontent.com/541093/186264642-f368b137-cac7-40d7-835b-d8a91a153ebb.mp4

**Comment by jasmussen** [comment]
Lorem your _ipsum_ 👌 

If you're feeling good on the semantic side, it seems like we have a good foundation in place. CC: @critterverse for some additional context on past explorations. Thank you @ryelle!

**Comment by thetinyl** [comment]
Thanks for setting this up @jasmussen!

I definitely agree with @ryelle that the whole line should be swapped. It opens up more opportunities for us to explore. 

I see two possible directions here, based on the feedback received:

**Option 1: Empowerment of ideas**
This fits most in line with the original concept behind "Publish your passion". That WordPress is more than the literal things/types of websites or projects you can create with it. 

The rotating headlines would be:

> WordPress: Publish your passion
> ": Launch your business
> ": Share your vision
> ": Grow your story

**Option 2: Empowerment of actions**
This direction is more explicit in calling out some literal ways WordPress can do it all. It does feel somewhat limiting (unless we can continuously add things to actually name them all), but might be quicker to understand across audiences. 

The rotating headlines would be:

> WordPress: Publish your passion
> ": Grow your business
> ": Launch your podcast
> ": Share your portfolio

Overall, I still feel like "Publish your passion" should stay as the main headline. With "publish", I think it speaks to the history of the platform, it's open source mission, and hope for the future. With "passion", I think it opens up the door for all kinds of folks (users, contributors, and extenders) to interpret it in a way that's most meaningful for them. Whether it's in reference to a user's hobby blog, an entrepreneur's driving force, or the way a contributor feels about being involved in such a strong community. Seems in line with the spirit of jazz to me.

**Comment by critterverse** [comment]
Nice, thanks for getting this rolling everyone!

One option we looked at in design explorations was "WordPress: your tool to _______" (build, experiment, etc).

But I like the idea of swapping the full phrase too! @thetinyl, out of the two options above, the empowerment of actions direction feels the most on brand to me although I would prefer that we use those lines _in place of_ "Pubish your passion" (not in addition to).

One way to think about this could be for each of the rotating lines to be a nod to a different WordPress audience. While I do think that we should try to speak to new users (and well, everyone) with this headline, it would be super cool to also speak more directly to those who have made WordPress into what it is today: IMO this is largely open-minded, tinkering-types of folks who don't get frustrated with small technical barriers-to-entry, and enjoy the self-discovery and endless customization options associated with the WordPress platform. Personally, I don't feel that the "Publish your passion" line really speaks to these folks, who are often developers that build and maintain sites for others and/or use WordPress for their day jobs (not necessarily just their hobbies/passions).

I wonder if there's a way to tailor these lines even further to make them feel more ownable to WordPress (and into something that couldn't possibly be used by a competitor). For example, there are a lot of platforms you could use to launch your podcast — but WordPress is democratizing publishing and known to be a platform with unbridled creative possibilites for self-expression.

Another format that seems to work well is the "The freedom to _____" used on the [About page](https://wordpress.org/about/), and I could see this living on the homepage instead. I like how the repetition of the word "freedom" emphasizes some of WordPress' stronger selling points of flexibility, extendibility, being open-source etc:

- WordPress: The freedom to build
- WordPress: The freedom to change
- WordPress: The freedom to share

^ It also feels big and aspirational in the same way that our mission of democratizing publishing does.

TL;DR: I think the goal should be to capture the attention both of new audiences/novice users while also being reflective of what makes the platform so special and unique ✨

**Comment by eidolonnight** [comment]
> While I do think that we should try to speak to new users (and well, everyone) with this headline, it would be super cool to also speak more directly to those who have made WordPress into what it is today: IMO this is largely open-minded, tinkering-types of folks who don't get frustrated with small technical barriers-to-entry, and enjoy the self-discovery and endless customization options associated with the WordPress platform. Personally, I don't feel that the "Publish your passion" line really speaks to these folks, who are often developers that build and maintain sites for others and/or use WordPress for their day jobs (not necessarily just their hobbies/passions).

While I understand this sentiment, I do want to note a section from our [philosophy](https://wordpress.org/about/philosophy/) that I think should guide our decision here (emphasis is my own):

> **Design for the Majority**
Many end users of WordPress are non-technically minded. They don’t know what AJAX is, nor do they care about which version of PHP they are using. **The average WordPress user simply wants to be able to write without problems or interruption. These are the users that we design the software for as they are ultimately the ones who are going to spend the most time using it for what it was built for.**

Of all the places for us to demonstrate this philosophy, the first section of the homepage seems like an important one.

**Comment by thetinyl** [comment]
Honestly, I had all these variations (and even more) in draft as previous options:

> WordPress: The freedom to build
> WordPress: The freedom to change
> WordPress: The freedom to share

Until I was looking at the Wix homepage where they use "freedom" (or very similar) a few times and it no longer felt ownable to WordPress (at least not for the homepage). 😅

**Comment by jpantani** [comment]
Since the homepage is intended as a gateway for top-of-funnel visitors, it’s important to convey the WordPress brand promise which is a message of empowerment: “To provide every person and entity who wants it, a web presence and or economic enablement online,” as well as its core DNA that WordPress is a publishing platform.

I think it's a tough task to accomplish this while remaining aspirational. Expanding the concept of the empowerment of ideas not actions like everyone else who has a website builder is crucial. I’d like to bring back to the forefront these headings:

> The rotating headlines would be:
>
> WordPress: Publish your passion
> “: Launch your business
> “: Share your vision
> “: Grow your story

There are other empowering messages to consider like:

- Unleash your creativity
- Push your boundaries
- Craft your masterpiece / Code your masterpiece


These all speak to audiences who are looking to use WordPress as a tool for business or for fun, whether a fulltime developer running an agency or a hobby blogger curating their recipes. As soon as we get specific, we exclude others by not showing an exact use case and folks start to feel as though they’re not represented in the messaging and move on.

**Comment by critterverse** [comment]
>While I understand this sentiment, I do want to note a section from our [philosophy](https://wordpress.org/about/philosophy/) that I think should guide our decision here

Thanks for sharing this link @eidolonnight! I agree that it makes sense to prioritize speaking to the average user who just needs to write without interruption. But it feels like by _only_ doing that, we're not really conveying the essence of the project and what makes it special. Also agree with @thetinyl about it being tricky to find a differentiating message from others making similar claims. But Wix is no WordPress — so how can we make that clear?

[This post](https://ma.tt/2021/04/wix-dirty-tricks/) from Matt Mullenweg about Wix’s WordPress attack ads last year seems relevant:

>WordPress is a global movement of hundreds of thousands of volunteers and community members, coming together to make the web a better place. The code, and everything you put into it, belongs to you, and its open source license ensures that you’re in complete control, now and forever. WordPress is free, and also gives you freedom.

WordPress is for everyone but it’s also so much bigger than any one user’s personal blog/online store/etc. To me, WordPress has always encapsulated the self-guided, DIY spirit of the early days of blogging while also being a shining beacon of a bright future — a cornerstone of the free and open internet. It's a community of novice and expert contributors coming together from all over the world to build this special thing that powers nearly half the web. It's a career path, the foundation to many thriving businesses, a vehicle for self-expression, and much more. 

I think there are ways to contextualize some of our differentiators, even/especially for novice users looking for a publishing platform! But agree with everyone else that it's hard to put into words so apologies for not having more concrete copy suggestions 😂

**Comment by josephahaden** [comment]
Great discussion here, folks. What I see as a common thread in all these comments are 1) the need to speak to our many audiences and 2) the need to connect each audience to the myriad uses of WordPress. 

I’m seeing words like “empowerment”, “aspirational” and “bigger than”. I am also seeing our audience words in here! Things like “self-guided” and  “every person” with some clear ties to entrepreneurs and WP service providers. These line up pretty tidily into three user paths and, I believe, are applicable to any type/level of skill: 

- People who have an idea and just need tools that make it possible (empowerment and self-guided)
- People who know their direction and just need a space to explore their idea (aspirational and every person)
- People who are looking to be part of something, tools negotiable (bigger than and self-expression)

**Based on these journeys (though not taking into account any need to be quirky), let's give these suggestions a go for the current iteration of this**:

1. Grow your business
2. Push your boundaries
3. Publish your passion (or "Share your vision" if this is nonnegotiable)


_Side Note: "Empowered [builders, creatives, agencies, audienceX], powered by WordPress" popped into my mind as I was working to synthesize all your thoughts. :) But also "quirky" things that came to mind were "like lunchmeat—always ready", "fast as a rat in a cracker box" and "used on purpose and on accident the world over". You can ignore all of this, I just had to tell someone._

**Comment by jasmussen** [comment]
Good thoughts all around. I would echo Channing's voice, that _openness_ feels like a value that sets apart and embodies WordPress, so worth having as a throughline. I'm also still excited about the mentions of adding a little quirkiness to balance it all out.

**Comment by jasmussen** [comment]
To elaborate a bit, I appreciate in Channing's suggestions specifically the inclusion of the word "Freedom" as a key value, from the GPL itself all the way through to the ability to migrate to and from the platform. Those are values WordPress can own, but many alternatives in the space cannot.

Josepha's three suggestions very well capture the listed journeys, as well as the the conversation. Could we try one of the suggestions from @thetinyl — "Freedom to build" — as one of the three? Perhaps instead of "Push your boundaries"?

**Comment by thetinyl** [comment]
Thanks for your thoughts @josephahaden! The breakdown helps frame up the direction we all seem to be heading. 

@jasmussen I think you're onto something with adding the "freedom" in somewhere. As much as I was uncomfortable with it as the main conceptual link, I think including it _among_ the other headlines changes its context from the way Wix uses it on their homepage. I'm not sure "Freedom to build" makes sense with the flow of the other headlines (e.g. [Blank] your [something]), BUT what about this:

1. Grow your business
2. Publish your passion
3. **Flex your freedom**

"Flex" being a nod to the flexibility of the platform (empowerment, self-expression), and recognizing WordPress as a skillful way to create that anyone can partake in (aspirational, self-guided, every person). "Freedom" being, well, freedom as it exists in the project, mission, and beyond (bigger than).

_As an aside_: @jasmussen I'm really curious about the "quirkiness" you mentioned. Would you be open to sharing some examples of existing language from elsewhere in the project that fits this idea? It would make for some helpful reference material.

**Comment by jasmussen** [comment]
Dig your thoughts 👌

In the original idea, we could get a little quirky in a grounding sense with the "Publish your ____" variant: something to balance out potentially high-flying language. [The P2 website is a good example](https://wordpress.com/p2/) where it kind of works because there's a wider range of options it cycles through, with "collective brain" following right after "shared workspace". Subtle, but just off-beat enough to potentially raise an eyebrow.

**Comment by josephahaden** [comment]
Quirky as in "casual and charming" not "weird" probably. For instance, under no circumstances should we reference lunchmeat or rats on the homepage as I shared above.  ;)

**Comment by critterverse** [comment]
Quick update that we've been running into one issue with the implementation that's been discussed here so far in https://github.com/WordPress/wporg-main-2022/pull/107. The various headlines break into either 2 or 3 lines of text across various screen sizes and across languages in a way that's making the layout difficult to control consistenty — we haven't found a way to ensure that the content is not jumping around on the page as the headline rotates (see https://github.com/WordPress/wporg-main-2022/pull/107#issuecomment-1230495599). 

I think we should give a different approach a try using a static headline, but with a different variation of the headline being used on each page load. It seems like the headlines work well in a standalone format too, but wanted to double check here in case this new implementation would require some copy tweaking! cc @thetinyl

**Comment by thetinyl** [comment]
Thanks for the update @critterverse! 

If there aren't any further spacing or line breaking issues with this alternative approach, then copy should be good to go! 👍 

Even in an animated rotation, there's no guarantee that someone landing on the page wouldn't scroll/navigate away before seeing more than one headline, so they should be able to resonate independently.
